test/core/surface/sequential_connectivity_test.cc

```diff
@@ -36,16 +37,15 @@
 typedef struct test_fixture {
   const char* name;
   void (*add_server_port)(grpc_server* server, const char* addr);
-  grpc_channel* (*create_channel)(const char* addr);
+  grpc_channel* (*create_channel)(const char* addr,
```

Comment:
This doesn't seem to be needed, since both functions that are plugged in here are identical, and both simply call `create_test_channel()`.  I think we can just have `run_test()` call `create_test_channel()` directly.